OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +30,53 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+
+		[Desc("Space between lines as a percentage (default) of line height or fixed pixel amount.")]
+		public int LineSpacing = 140;
+
+		[Desc("Percentage: line height = LineSpacing% * font size.",
+			"FixedMargin: line height = font size + LineSpacing.",
+			"FixedHeight: line height = LineSpacing.")]
+		public LineSpacingType LineSpacingType = LineSpacingType.Percentage;
+		public LineVAlign LineVAlign = LineVAlign.Middle;
```

Comment:
Both LibreOffice Writer and Calligra Words put the text at the top of the line (don't move first line text based on line spacing).